src/Map.ts

```diff
@@ -219,6 +221,7 @@ export class Map extends maplibregl.Map {
   private curentProjection: ProjectionTypes = undefined;
   private originalLabelStyle = new window.Map<string, ExpressionSpecification | string>();
   private isStyleLocalized = false;
+  private languageIsUpdated = false;
```

Comment:
How do you feel about using getter for that? I would be like:
```
/*
 * explanation
 */
private _isLanguageUpdated = false;
public get isLanguageUpdated() {
  return this._isLanguageUpdated;
}
```

Reply:
We could sure do that, but then why not use getters for all the get-kinds of method in Map like `.isGlobeProjection()`, `.getMaptilerSessionId()`, and many more. I have nothing against getters/setters (i actually think they are pretty cool) but it's not something I've been using in this project because Maplibre is not using them either so out of consistency I think it's better to avoid them, at least for the Map class. Using them alongside `.getSomtehing()` methods would add confusion and lead users to think they are reaching properties instead of getters.